bin/add-roles.sh

```diff
@@ -12,7 +12,7 @@
 ./bin/utils/ccd-add-role.sh "RES-SOL-ONE-SPEC-PROFILE"
 ./bin/utils/ccd-add-role.sh "RES-SOL-TWO-UNSPEC-PROFILE"
 ./bin/utils/ccd-add-role.sh "RES-SOL-TWO-SPEC-PROFILE"
-./bin/utils/ccd-add-role.sh "basic-access"
+./bin/utils/ccd-add-role.sh "ga-basic-access"
```

Comment:
@deepthidoppalapudihmcts  has this role been created in demo and preview?

Reply:
Hi Karthick, I have created this user in demo and added this role to the script in civil-sdk